fs/assets/credits_generated/github.com/hashicorp/hcl/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/hashicorp/hcl
 
+go 1.22
```

Comment:
to fix this issue https://github.com/GoogleContainerTools/skaffold/pull/9352#issuecomment-2053899682